providers/zendesksupport/metadata/schemas.json

```diff
@@ -1,6 +1,137 @@
 {
   "modules": {
-    "": {
+    "help-center": {
+      "objects": {
+        "article_labels": {
+          "displayName": "Article Labels",
+          "urlPath": "/api/v2/help_center/articles/labels",
```

Comment:
Can we pull `/api/v2/` out of modules into the connector itself? If the version ever changes, we'd have to make the change in multiple places for it to take effect. If we have a structure from my comment in #1049 where we can specify multiple properties on the module (path, etc), then the version could be a part of the path. 

If we need a specific object to be a different version, we could add the ability to override the path at an object level (we don't need to add that now, but just to make sure this approach works)